src/index.js

```diff
@@ -15,3 +15,5 @@ process.on('SIGTERM', () => {
     logger.info('Http server closed.');
   });
 });
+
+export default server;
```

Comment:
Nice work but I think you should remove this one because there is no need for exporting the server